hypothesis-python/src/hypothesis/internal/conjecture/utils.py

```diff
@@ -30,6 +30,7 @@
 from hypothesis.internal.floats import int_to_float
 
 LABEL_MASK = 2 ** 64 - 1
+_SEQUENCE_TYPES = (OrderedDict, Sequence, enum.EnumMeta)
```

Comment:
I'd delete this and just use it inline.

Reply:
I'll revert it, but there's definitely some value in abstracting it out.